create-prs-to-update-vcs-repositories/create_prs_to_update_vcs_repositories.py

```diff
@@ -131,29 +135,13 @@ def parse_args() -> argparse.Namespace:
     args_repo.add_argument("--base_branch", type=str, default="main", help="The base branch of autoware.repos")
     args_repo.add_argument("--new_branch_prefix", type=str, default="feat/update-", help="The prefix of the new branch name")
 
-    '''
-    Following default pattern = r'\b(v?\d+\.\d+(?:\.\d+)?(?:-\w+)?(?:\+\w+(\.\d+)?)?)\b'
-    can parse the following example formats:
-        "0.0.1",
-        "v0.0.1",
-        "ros2-v0.0.4",
-        "xxx-1.0.0-yyy",
-        "2.3.4",
-        "v1.2.3-beta",
-        "v1.0",
-        "v2",
-        "1.0.0-alpha+001",
-        "v1.0.0-rc1+build.1",
-        "2.0.0+build.1848",
-        "2.0.1-alpha.1227",
-        "1.0.0-alpha.beta",
-        "ros_humble-v0.10.2"
-    '''
+    # Define an argument to specify which version components to check
     args_repo.add_argument(
-        "--semantic_version_pattern",
-        type=str,
-        default=r'\b(v?\d+\.\d+(?:\.\d+)?(?:-\w+)?(?:\+\w+(\.\d+)?)?)\b',
-        help="The pattern of semantic version"
+        '--targets',
+        choices=['major', 'minor', 'patch', 'any'],  # Restrict choices to 'major', 'minor', 'patch', and 'any'
```

Comment:
Add global variable `VALID_RELEASES = ['major', 'minor', 'patch', 'any']` somewhere and then change to

```suggestion
        choices=VALID_RELEASES,  # Restrict choices to 'major', 'minor', 'patch', and 'any'
```

Reply:
(by [this commit](https://github.com/autowarefoundation/autoware-github-actions/pull/320/commits/6fb60c86534cb4e1f2a68db9df84bba14ba64438))